app/controllers/subscriptions_controller.rb

```diff
@@ -135,4 +133,22 @@ def get_subscriber_email(subscriber_id)
   def add_top_level_breadcrumbs
     add_breadcrumb 'Subscriptions', subscriptions_path
   end
+
+  def fetch_subscription
+    permitted_params = params.permit(:id, :revision_id)
+    @concept_id = permitted_params['id']
+
+    subscription = get_latest_revision(@concept_id, permitted_params['revision_id'] || 0)
+
+    if subscription
```

Comment:
i would suggest using `fetch` in here as well

Reply:
done